### ISSUE TITLE
Pudge hook sound issue.

### ISSUE THREAD

**Issue by X-NightCrowLS** [ValveSoftware/Dota-2#2212]
#### Your system information

* System information from steam (`Steam` -> `Help` -> `System Information`) in a [gist](https://gist.github.com/): //
* Have you checked for system updates?: [Yes/No] Yes
* Are you using the latest stable video driver available for your system? [Yes/No] Yes
* Have you verified the game files?: [Yes/No] Yes

#### Please describe your issue in as much detail as possible:

Basically when you hit an enemy with your hook, it doesn't reproduce the right sound but instead, it reproduces the "Missed hook sound" so sometimes is hard to know if i get someone from the fog of war.

#### Steps for reproducing this issue:

1.  Play pudge
2. Hook an enemy either with normal or arcana skin.
